### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by hellendag**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+* Create Slack team inviter
+* Export `KeyBindingUtil` and `getDefaultKeyBinding`
+* Website autobuild, Travis/NPM badges
+
+### Fixed
+
+* Triple-clicks followed by block type changes now only affect first block
+* `DraftEditorLeaf` now re-renders correctly when its styles change
+* Backspace behavior within empty code blocks
+* Documentation typos, broken examples
```
Comment (zpao) [inline CHANGELOG.md]: Probably don't need to call this out (same with the website / travis stuff above). That's all pretty fluid and happens out of sync with releases.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## 0.2.0 (March 8, 2016)
+
+### Added
+
+* Move `white-space: pre-wrap` into inline style to resolve insertion issues
+* `handleDrop` prop method for `Editor` to allow manual drop management
+* `decoratedText` prop for decorator components
+* `getVisibleSelectionRect`, to provide Rect for DOM selection
+* Create Slack team inviter
```
Comment (spicyj) [inline CHANGELOG.md]: I don't normally note non-source changes in changelogs but if you want to this also seems fine.

**Comment by spicyj** [comment]
:+1:

**Comment by hellendag** [comment]
@facebook-github-bot shipit

**Comment by mitermayer** [comment]
sweet!!

**Pull request closed without merging**